### ISSUE TITLE
F-Droid can't build

### ISSUE THREAD

**Issue by licaon-kter** [jfoclpf/in-my-district#51]
```
before_prepare : hooks/generateJsFiles.js
ENOENT: no such file or directory, open '/home/vagrant/build/com.in.my.district/keys-configs/configs.json
```

ref: https://web.archive.org/web/20230124090902/https://monitor.f-droid.org/builds/log/com.in.my.district/10228

/LE: https://gitlab.com/fdroid/fdroiddata/-/commit/7a14b8b452e7a8fd14f16025e60b9ad160f6d4df

**Comment by jfoclpf** [comment]
Thanks for pinging

The env variable `CI` should be set to `true`.

It will use another standard (public) configuration file but without the credentials of the DB (no need for the app). This happens because the repo is a monorepo with the server code included and the repo has a single configuration file.

**Comment by jfoclpf** [comment]
TLDR, in Bash

`CI=1`

**Comment by licaon-kter** [comment]
Added thanks
https://gitlab.com/fdroid/fdroiddata/-/commit/1a8a88900274922d7d87631e4cd55e3243148287
